app/views/cclow/home/_latest_additions.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="slider-container">
+  <div class="slider" data-slick='{"slidesToShow": 3, "slidesToScroll": 3}'>
+    <% @latest_additions.each do |item| %>
+      <div class="item-container">
+        <div class="item">
+          <div class="additions-kind"><%= item[:kind] %></div>
+          <div class="title"><a href="#"><%= item[:title] %></a></div>
+          <div class="meta">
+            <img class="flag" src=<%= asset_path "flags/#{item[:iso]}.svg"%> alt="">
+            <div><%= item[:jurisdiction] %>  |  <%= item[:addition_type] %>  |  <%= item[:date_passed] %></div>
+          </div>
+        </div>
+      </div>
+    <% end %>
+  </div>
+</div>
+
+
+<%= javascript_tag do %>
```

Comment:
For consistency and readability reasons I'd move html + js from this file to react component and render it here like all the other ones:
```
<%= react_component("YourSliderComponeny", { your props }) %>
```
you also need to add `onClick` functionality to the slider box

Reply:
Hmm, not sure if we need react for that simple carousel.

Reply:
it's a slick slider.

Reply:
Could be also done with stimulus `slider_controller`

```
 <div class="slider" data-controller='slider-controller' data-slick='{"slidesToShow": 3, "slidesToScroll": 3}'>
```

In `shared/controllers/slider_controller.js`
```
import { Controller } from 'stimulus';

export default class extends Controller {
  connect() {
    this.element.slick({
      prevArrow: '<button type="button" class="slick-prev"><i class="fa fa-1x fa-arrow-left"></i></button>',
      nextArrow: '<button type="button" class="slick-next"><i class="fa fa-1x fa-arrow-right"></i></button>'
    });
  }
  
  disconnect() {
     this.element.unslick();
  }
}
```
Something like this.

Reply:
Something similar I would do with tooltips.

Reply:
There's also the testimonials (quotes?) component that should be fairly simple to adapt to this scenario.